tests/maintel/test_random_walk.py

```diff
@@ -0,0 +1,84 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import logging
+import unittest
+
+from lsst.ts import externalscripts, standardscripts
+from lsst.ts.externalscripts.maintel import RandomWalk
+from lsst.ts.idl.enums import Script
+
+
+class TestRandomWalk(
+    standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase
+):
+    def setUp(self):
+        self.log = logging.getLogger(__name__)
```

Comment:
One thing to keep in mind is that ``setUp`` is called before every test. Maybe use ``setUpClass`` instead? it would look something like this:

```
    @classmethod
    def setUpClass(cls) -> None:
        cls.log = logging.getLogger(__name__)
        cls.log.propagate = True
```

But this is a personal preference, to avoid repeating this for every test.